csm_web/scheduler/models.py

```diff
@@ -95,7 +96,7 @@ class Meta:
 class Student(Profile):
     """
     Represents a given "instance" of a student. Every section in which a student enrolls should
-    have a new Mentor profile.
```

Comment:
Just fixing some random typos I noticed, not related to the functionality added in this PR